Cargo.toml

```diff
@@ -12,7 +12,8 @@ edition = "2018"
 
 [dependencies]
 atty = "0.2"
-error-chain = "0.12"
+color-eyre = "0.6.1"
+eyre = "0.6.7"
```

Comment:
```suggestion
color-eyre = "0.6"
eyre = "0.6"
```

Reply:
I don't like being non-specific about minimal versions, but we're a bin target and people should install with `--locked`, so I guess this is fine